src/applications/vaos/utils/constants.js

```diff
@@ -18,6 +18,7 @@ export const APPOINTMENT_STATUS = {
   cancelled: 'cancelled',
   fulfilled: 'fulfilled',
   noshow: 'noshow',
+  hideStatus: 'hideStatus',
```

Comment:
Could we set `status` to null instead of creating a new `hideStatus` status? I think `hideStatus` is confusing, since it's not actually an appointment status.